pages/en/developer/graphql-api.mdx

```diff
@@ -134,9 +134,25 @@ You can use suffixes like `_gt`, `_lte` for value comparison:
 }
 ```
 
-Full list of parameter suffixes:
+#### Example for block filtering
+
+You can also filter entities by the `_change_block`:
```

Comment:
Do you think you could add a little more information here on what this means, and what it enables?
i.e. This filters entities which were updated in or after the specified block. This can be useful if you are looking to fetch only entities which have changed, for example since the last time you polled. Or alternatively it can be useful to investigate or debug how entities are changing in your subgraph (if combined with a block filter, you can isolate only entities that changed in a specific block)